packages/blaze-html-templates/.versions

```diff
@@ -17,7 +17,6 @@ htmljs@1.0.11
 id-map@1.0.8
 jquery@1.11.9
 meteor@1.6.0
-minifier-js@1.2.14
```

Comment:
But wouldn't this be removed automatically, once a package is released? `meteor publish` will regenerate `.versions` and remove `minifier-js` because packages do not depend anymore on it?